pkg/api/node/formatter.go

```diff
@@ -207,3 +238,84 @@ func (h ActionHandler) listUnhealthyVM(rw http.ResponseWriter, node *corev1.Node
 func (h ActionHandler) maintenancePossible(node *corev1.Node) error {
 	return drainhelper.DrainPossible(h.nodeCache, node)
 }
+
+func (h ActionHandler) powerActionPossible(rw http.ResponseWriter, node string) error {
+	addon, err := h.addonCache.Get(defaultAddonNamespace, seederAddonName)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			logrus.Errorf("addon %s not found", seederAddonName)
+			rw.WriteHeader(http.StatusNotFound)
```

Comment:
Perhaps `StatusFailedDependency` is more appropriate, since Node resource is found.

Reply:
rebased and updated header.